lua/nvim-tree.lua

```diff
@@ -587,7 +586,10 @@ local function validate_options(conf)
     for k, v in pairs(user) do
       local invalid
       local override_typecheck = FIELD_OVERRIDE_TYPECHECK[k] or {}
-      if def[k] == nil then
+      if type(v) == "table" and not next(v) then
```

Comment:
please review this; validation is fragile

Reply:
why this change ? I don't see how it relates to this PR

Reply:
This was for the `update_to_buf_dir` case. It would be an empty table after `move_missing_val`.

Moved to legacy : manually remove that table.